src/test/techdebt.test.ts

```diff
@@ -23,4 +23,13 @@ describe('tech debt', function () {
             'remove filesystemUtilities.findFile(), use vscode.workspace.findFiles() instead'
         )
     })
+
+    it('nodejs minimum version', async function () {
+        const minNodejs = env.getMinNodejsVersion()
+
+        assert.ok(
+            semver.lt(minNodejs, '16.0.0'),
+            'remove require("perf_hooks").performance workarounds, use globalThis.performance instead (always available since nodejs 16.x)'
+        )
+    })
```

Comment:
We could start using `AbortController` too

Reply:
Kind of? `AbortController` becomes the 'official' way to cancel things related to Node. Like terminating a child process is a lot cleaner. Same for HTTP requests.